src/main/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2.java

```diff
@@ -94,6 +95,9 @@ public class SnowflakeSinkServiceV2 implements SnowflakeSinkService {
    */
   private final Map<String, TopicPartitionChannel> partitionsToChannel;
 
+  // Cache for schema evolution
+  private Map<String, Boolean> tableName2SchemaEvolutionPermission = new HashMap<>();
```

Comment:
You don't need to initialize here if you choose to do it in the constructor

Reply:
Modified